sunpy/visualization/imageanimator.py

```diff
@@ -878,3 +880,29 @@ def update_plot(self, val, line, slider):
         if val != slider.cval:
             line.set_ydata(self.data[self.frame_slice])
             slider.cval = val
+
+class ImageAnimatorWCS(ImageAnimator):
+    """
+    Same parameters as ImageAnimator with a extra parameter for WCS.
+    """
```

Comment:
Can you just and paste the ImageAnimator parameters and add the wcs parameter?  So the docstring stands alone.